packages/vscode-extension/src/debug/depsChecker/vscodeAdapter.ts

```diff
@@ -97,6 +97,10 @@ export class VSCodeAdapter implements IDepsAdapter {
     );
     return configuration.get<boolean>(key, false);
   }
+
+  private static async openUrl(url: string): Promise<void> {
+    return commands.executeCommand("vscode.open", Uri.parse(url));
```

Comment:
`async` and `await` should be paired?

Reply:
Just a wrapper, I think open url should be a async method.